pages/common/npx.md

```diff
@@ -3,21 +3,21 @@
 > Execute binaries from `npm` packages.
 > More information: <https://github.com/npm/npx>.
 
-- Execute the binary from a given npm module:
+- Execute the command from a local or remote `npm` package:
 
-`npx {{module_name}} {{command_arguments}}`
+`npx {{command}} {{command_arguments}}`
```

Comment:
```suggestion
`npx {{command}} {{arg1 arg2 ...}}`
```

Mandatory fix: [ellipsis must be used](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases).